build.sbt

```diff
@@ -67,11 +67,6 @@ lazy val core = project
       "ws.unfiltered" %% "unfiltered-filter" % unfilteredVersion,
       "ws.unfiltered" %% "unfiltered-jetty" % unfilteredVersion,
       "ws.unfiltered" %% "unfiltered-specs2" % unfilteredVersion % Test,
-    ).map(
```

Comment:
@mkurz This is causing a regression, see https://github.com/apache/pekko-persistence-cassandra/actions/runs/8405372473/job/23018005449?pr=176

Reply:
Sorry! Seems this is now fixed via
- #217